datajoint/base_relation.py

```diff
@@ -177,7 +177,7 @@ def make_placeholder(name, value):
                     value = pack(value)
                     placeholder = '%s'
                 elif heading[name].numeric:
-                    if value is None or np.isnan(np.float(value)):  # nans are turned into NULLs
+                    if value in [None, ''] or np.isnan(np.float(value)): # nans are turned into NULLs
```

Comment:
I agree with the change.  However, I feel that the condition should rather be
```python
value is None or value == '' or np.isnan(np.float(value))
```
and not rely on `__contains__` to work correctly with `None`.  It is probably more efficient too.